src/sentry/static/sentry/app/actionCreators/modal.jsx

```diff
@@ -60,6 +60,20 @@ export function openDiffModal(options) {
     });
 }
 
+/**
+ * @param Object options
+ * @param Object options.organization The organization to create a team for
+ */
+export function openCreateIncidentModal(options = {}) {
+  import(/* webpackChunkName: "CreateIncidentModal" */ '../components/modals/createIncidentModal')
```

Comment:
```suggestion
  import(/* webpackChunkName: "CreateIncidentModal" */ 'app/components/modals/createIncidentModal')
```
To match your other pull request?